sklearn/mixture/gmm.py

```diff
@@ -64,7 +64,8 @@ def log_multivariate_normal_density(X, means, covars, covariance_type='diag'):
     return log_multivariate_normal_density_dict[covariance_type](
         X, means, covars)
 
-
+@deprecated("The class function sample_gaussian is deprecated and "
+            "will be removed in 0.20. Use numpy.multivariate_normal instead.")
```

Comment:
Looking at the logs, there is only one test which uses it, `test_sample_gaussian`.